<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimersEdit</class>
 <widget class="QMainWindow" name="TimersEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/48x48/timer.png</normaloff>:/icons/48x48/timer.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMain">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="TimersDisplayWidget" name="timersWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageNew">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="buttonCreate">
          <property name="text">
           <string>Create</string>
          </property>
          <property name="icon">
           <iconset resource="../../../data/icons.qrc">
            <normaloff>:/icons/24x24/add.png</normaloff>:/icons/24x24/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutName">
          <item>
           <widget class="QLabel" name="labelNameNew">
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="editNameNew"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelNew">
          <property name="text">
           <string>Select a channel from the list below:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PlaylistDisplayWidget" name="playlistWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
  </widget>
  <widget class="QDockWidget" name="editor">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>318</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/timer.png</normaloff>:/icons/24x24/timer.png</iconset>
   </property>
   <property name="windowTitle">
    <string>Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelChannel">
       <property name="text">
        <string>Channel:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPlaylist">
       <property name="text">
        <string>Playlist:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelStartTime">
       <property name="text">
        <string>Start time:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelEndTime">
       <property name="text">
        <string>End time:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="editDate">
       <property name="date">
        <date>
         <year>2010</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QTimeEdit" name="editStartTime">
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTimeEdit" name="editEndTime">
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelDisabled">
       <property name="text">
        <string>Disabled:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkBoxDisabled"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editUrl">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelUrl">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="editType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Once</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Daily</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weekly</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="layoutChannel">
       <item>
        <widget class="QLCDNumber" name="editNum">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="numDigits">
          <number>3</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editChannel">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editPlaylist">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="labelInfo">
       <property name="text">
        <string>&lt;i&gt;Timers must not overlap! This will be automatically checked in the future.&lt;/i&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/add.png</normaloff>:/icons/24x24/add.png</iconset>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/remove.png</normaloff>:/icons/24x24/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/save.png</normaloff>:/icons/24x24/save.png</iconset>
   </property>
   <property name="text">
    <string>Save and Close</string>
   </property>
   <property name="toolTip">
    <string>Save and Close</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../../data/icons.qrc">
     <normaloff>:/icons/24x24/close.png</normaloff>:/icons/24x24/close.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistDisplayWidget</class>
   <extends>QWidget</extends>
   <header>ui/playlist/PlaylistDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimersDisplayWidget</class>
   <extends>QWidget</extends>
   <header>ui/recorder/TimersDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections/>
</ui>
